### ISSUE TITLE
Status: 500. Message: db query error: pq: column ti.execution_date does not exist

### ISSUE THREAD

**Issue by ghost** [lopesdiego12/Grafana#1]
Hi,

None of the panels are showing any data. The error is:

```
Status: 500. Message: db query error: pq: syntax error at or near ")"
```

Airflow version: v2.3.3
Postgres version: PostgreSQL 13.7 on x86_64-pc-linux-gnu, compiled by gcc (GCC) 7.3.1 20180712 (Red Hat 7.3.1-12), 64-bit
Grafana version:

**Comment by lopesdiego12** [comment]
Hi Hutch,

You have to edit for your proper postgres connection.

1 - Click Connections in the left-side menu.

2 - Under Your connections, click Data sources.

3 - Enter PostgreSQL in the search bar.

4 -Select PostgreSQL.

5 - The Settings tab of the data source is displayed.

6 - Set the data source’s basic configuration options to your postgres connection

7 - Test connection and then it will works

**Comment by ghost** [comment]
Thanks for the quick turn around. I can now successfully connect to my airflow_db but I think newer airflow versions have changed the column names.

```
Status: 500. Message: db query error: pq: column "execution_date" does not exist
```

**Comment by lopesdiego12** [comment]
Yeah, you are right, this template totally match with <2 airflow version.
You can take a look at https://airflow.apache.org/docs/apache-airflow/stable/database-erd-ref.html this is updated ERD reference for all table on apache airflow 2.0.
I think the concept is the same, have data pipelines and environment health checks.
Sorry this template never more got updated but after you do that you can upload the new template.
If you want we can discuss together in a call

**Comment by ghost** [comment]
Would it be possible for you to update the code? Or is this project no longer being updated? Thanks

**Comment by samush** [comment]
> Would it be possible for you to update the code? Or is this project no longer being updated? Thanks

This nice man [did](https://github.com/hw0603/MultiCloud-Platform/blob/main/grafana/provisioning/dashboards/airflow_monitoring.json) this job

Also, because i don't know Korean language for me personally it's more comfortable to replace this words with English ones,
[airflow_monitoring_upd.json](https://github.com/user-attachments/files/15586692/airflow_monitoring_upd.json)
